src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <p>{{ title }}</p>
```

Comment:
I would use a <h*> tag for the title but not sure about what are your requirements